test/web.dart

```diff
@@ -6,7 +6,6 @@ library dart_pad.web_test;
 
 import 'core/dependencies_test.dart' as dependencies_test;
 import 'core/keys_test.dart' as keys_test;
-import 'documentation_test.dart' as documentation_test;
```

Comment:
Should we also delete `documentation_test.dart` in this PR?

Reply:
He already did so and I think it's fine to remove it.